src/Maestro/DependencyUpdateErrorProcessor/Program.cs

```diff
@@ -0,0 +1,99 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Reflection;
+using Azure.Core;
+using Azure.Identity;
+using Maestro.AzureDevOps;
+using Maestro.Data;
+using Microsoft.DncEng.Configuration.Extensions;
+using Microsoft.Dotnet.GitHub.Authentication;
+using Microsoft.DotNet.GitHub.Authentication;
+using Microsoft.DotNet.ServiceFabric.ServiceHost;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Octokit;
+
+namespace DependencyUpdateErrorProcessor
+{
+    internal static class Program
+    {
+        /// <summary>
+        /// This is the entry point of the service host process.
+        /// </summary>
+        private static void Main()
+        {
+            ServiceHost.Run(
+                host =>
+                {
+                    host.RegisterStatefulService<DependencyUpdateErrorProcessor>("DependencyUpdateErrorProcessorType");
+                    host.ConfigureServices(
+                        services =>
+                        {
+                            services.AddDefaultJsonConfiguration();
+                            services.AddBuildAssetRegistry(
+                                (provider, options) =>
+                                {
+                                    var config = provider.GetRequiredService<IConfiguration>();
+                                    options.UseSqlServer(config.GetSection("BuildAssetRegistry")["ConnectionString"]);
+                                });
+                            services.AddAzureDevOpsTokenProvider();
+                            services.Configure<AzureDevOpsTokenProviderOptions>(
```

Comment:
Can you remind me why we need an azure devops token provider here? Are we actually doing anything in Azdo?

Reply:
I think in some point I used the token provider. I think its no longer relevant. I will remove this.